apps/challenges/models.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, *args, **kwargs):
     published = models.BooleanField(
         default=False, verbose_name="Publicly Available", db_index=True
     )
+    can_participate = models.BooleanField(default=True)
```

Comment:
Can we please rename`can_participate` to `is_registration_open`?

Reply:
@RishabhJain2018 done